types/react/experimental.d.ts

```diff
@@ -129,4 +129,6 @@ declare module '.' {
     export type Usable<T> = Thenable<T> | Context<T>;
 
     export function experimental_use<T>(usable: Usable<T>): T;
+
+    export function experimental_useEvent<T extends (...args: any[]) => any>(callback: T): T;
```

Comment:
This should match `useCallback` so that we get a consistent behavior for implicit-any.